arctopSDK/README.md

```diff
@@ -6,7 +6,7 @@ Arctop's SDK repository contains everything you need to connect your application
 
 Arctop is a software company that makes real-time cognition decoding technology. Arctop's software applies artificial intelligence to electric measurements of brain activity — translating people’s feelings, reactions, and intent — into actionable data that empower applications with new capabilities. Since its founding in 2016, Arctop has worked to develop a cross-platform SDK that provides noninvasive brain-computer interface capabilities. The SDK is being developed continuously and is the result of  deep R&D, such as [this peer-reviewed study](https://www.frontiersin.org/articles/10.3389/fncom.2021.760561/full) published in _Frontiers in Computational Neuroscience_ where Arctop's SDK was used in a personalized audio application.
 
-The current version of the Arctop SDK provides four unique cognition data streams: focus, enjoyment, stress, and sleep. It also provides body data streams including eye blinks and heart rate. All data streams are provided in real-time, meaning applications receive new data points several times per second. 
+The current version of the Arctop SDK provides three unique cognition data streams: focus, enjoyment, flow, and sleep state. It also provides body data streams including eye blinks, heart rate, and heart rate variability. All data streams are provided in real-time, meaning applications receive new data points several times per second. 
```

Comment:
@ls-arctop 'focus, enjoyment, flow, and sleep state' shouldn't it still be four?

Reply:
Yes, thanks, good catch!